langchain/src/chat_models/base.ts

```diff
@@ -22,26 +23,22 @@ const getCallbackManager = (): LLMCallbackManager => ({
   },
 });
 
-const getVerbosity = () => true;
-
 export type SerializedChatModel = {
   _model: string;
   _type: string;
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
 } & Record<string, any>;
 
+export interface BaseChatModelParams extends BaseLanguageModelParams {
+  callbackManager?: LLMCallbackManager;
```

Comment:
`callbackManager` is present in both `BaseChatModelParams` and `BaseLLMParams`, let's move to `BaseChatModelParams`

Reply:
For sure, and a bunch of other things too. I didn't move it on purpose, as it sounded easier to do at the same time as you actually implement it.